changelog/docblock-column-mapper-set_defaults

```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: fix
```

Comment:
Do you usually mark docblock updates as a fix or tweak ? I would mark it as a tweak to be honest

Reply:
Changed